.github/tests/prometheus/pre-install.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+helm install kube-prometheus-stack kube-prometheus-stack --version 45.7.1 --repo https://prometheus-community.github.io/helm-charts -n "$scenario" --wait
```

Comment:
Worried about this version getting out of date and having to keep it up to date. Is there a latest tag we can use instead?

Reply:
I usually prefer to pin the versions in tests to ensure they don't break at random times and we spend a bunch of our time, or contributor time, trying to "fix a broken pr" when its the test that spontaneously breaks because upstream changed something. It lets us catch the breakage when we bump up the version of the chart in its own pr. The podMonitor api hasn't changed for ages so we don't need to keep on top of the bleeding edge of kube-prometheus-stack as well.

Reply:
Also, we already have versions pinned for things in the other tests...

That gives me an idea though... actions can run periodically. I could put the version numbers in its own file and add a job to check for updates and either let us know, or maybe even a pr, with updated versions.

Reply:
Here's a start of that idea: https://github.com/spiffe/helm-charts/pull/184